index.html

```diff
@@ -4052,6 +4026,37 @@ <h3>Securing Mechanism Specifications</h3>
 of the [[[?VC-SPECS]]] [[?VC-SPECS]].
         </p>
 
+        <p>
+Securing mechanism specifications that create new types of [=embedded proofs=]
+MUST specify a [=property=] for securing both [=verifiable credentials=] and
+[=verifiable presentations=]. The requirements for the property used by the
+embedded securing mechanism are as follows:
+        </p>
+
+        <ul>
+          <li>
+The property MUST relate the [=verifiable credential=] or [=verifiable
+presentation=] to a separable and securable [=proof graph=].
+          </li>
+          <li>
+The property MUST define all terms used by the [=proof graph=]. For example,
+the mechanism could define vocabulary specifications and `@context` files
+in the same manner as they are utilized by this specification.
+          </li>
+          <li>
+In a [=verifiable credential=], the property MUST secure the
+[=default graph=].
+          </li>
+          <li>
+In a [=verifiable presentation=], the property MUST secure the
+[=default graph=] of the [=presentation=] as well as every [=proof graph=]
+related to each [=verifiable credential=] in the [=presentation=].
+          </li>
+          <li>
+The property MAY reuse the `proof` property as defined in [[VC-DATA-INTEGRITY]].
```

Comment:
"Reuse" sounds like the data-integrity `proof` property would be used as a component of the securing mechanism's property, but I think you mean that it would just _be_ that property. Maybe

```suggestion
The property MAY be the `proof` property as defined in [[VC-DATA-INTEGRITY]].
```

or the more complicated "The embedded securing mechanism MAY use the `proof` property defined by [[VC-DATA-INTEGRITY]] as its securing property."

Reply:
```suggestion
The `proof` property as defined in [[VC-DATA-INTEGRITY]] MAY be used by the
embedded securing mechanism.
```